src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -168,6 +168,17 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
             NullableFlowState topLevelNullability,
             int position,
             SymbolDisplayFormat? format = null);
+
+        /// <summary>
+        /// Nullable annotation associated with the type, or <see cref="NullableAnnotation.None"/> array if there are none.
+        /// </summary>
+        NullableAnnotation NullableAnnotation { get; }
+
+        /// <summary>
+        /// Returns a type with the given nullable annotation.
```

Comment:
>a type [](start = 20, length = 6)

Presumably not just "a" type, but it returns the same type as this type but with a different nullable annotation.  Please clarify.